swap_meet/item.py

```diff
@@ -1,2 +1,21 @@
 class Item:
-    pass
\ No newline at end of file
+    def __init__(self, category="", condition=0, age=0):
+        self.category = category
+        self.condition = condition
+
+
+    def __str__(self):
+        return "Hello World!"
+
+
+    def condition_description(self):
+        descriptions = {
+            0: "You tell me?",
+            1: "Do you really want this?",
+            2: "This might be nice in the corner on a shelf somewhere.",
+            3: "It's better than nothing!",
+            4: "I used it but I didn't really use it",
+            5: "Just as good as the store but it'll cost you less",
+        }
```

Comment:
Nice data-driven approach. Since the keys are just ints, and even start from 0, we might consider using a list instead, though the explicit documentation of what score corresponds to which string is kind of nice. But notice that if we needed to expand the range of scores and put a new one in the middle, we'd need to shift everything down by one.

Also, even though there's no test to get a description for a fractional condition, there _is_  a test that shows we should expect to need to handle them (condition=3.5). What would happen in this situation (a situation we should expect)? Or what if something out of the expected condition range was set (a situation it might be OK for us _not_ to expect).